src/app/technical-record/vehicle-summary/vehicle-summary.component.ts

```diff
@@ -9,7 +10,7 @@ import { TechRecord } from '@app/models/tech-record.model';
 export class VehicleSummaryComponent implements OnInit {
   @Input() activeRecord: TechRecord;
   vehicleClassDescription = '-';
-  constructor() {}
+  constructor(public techRecHelpers: TechRecordHelpersService) {}
 
   ngOnInit() {
     this.vehicleClassDescription =
```

Comment:
This isn't needed anymore. There is a capitalizePipe. This was overlooked earlier should be fixed now.

Reply:
it is for `isStandardVehicleType` on the template

Reply:
are you talking about ngOnInit ?